lib/base/RequestClient.js

```diff
@@ -94,6 +95,15 @@ RequestClient.prototype.request = function(opts) {
   this.lastRequest = new Request(optionsRequest);
 
   axios(options).then((response) => {
+    if (opts.logLevel === "debug") {
+      parseRequest(optionsRequest)
+      console.log(`response.statusCode: ${response.status}`)
+      console.log(`response.headers: ${JSON.stringify(response.headers)}`)
+      if (response.status < 200 || response.status >= 400) {
```

Comment:
I think we do this test elsewhere and should not be testing here.

Reply:
Is there another location that should be outputting this error message or should this just do a quick check to see if `response.data.code || response.data.message || response.data.more_info` exists then print the error message?

Reply:
What about just logging `response.data`?